src/components/Constants/LoadConstants.test.tsx

```diff
@@ -0,0 +1,195 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { GqlMockedProvider } from '../../../__tests__/util/graphqlMocking';
+import { useApiConstants } from './UseApiConstants';
+
+describe('LoadConstants', () => {
+  it('query correct', async () => {
+    const { result, waitForNextUpdate } = renderHook(() => useApiConstants(), {
+      wrapper: GqlMockedProvider,
```

Comment:
If you wanted to test that the query was called with the correct args, you can use `onCall`.

To use `onCall` I think you need to create a new wrapper component so you can pass props to ergonomock. You can just do it inline:
```suggestion
      wrapper: ({ children }: { children: ReactElement }) => (
        <GqlMockedProvider onCall={onCall}>{children}</GqlMockedProvider>
      ),
```
https://github.com/SurveyMonkey/graphql-ergonomock#:~:text=Finally,%20you%20can%20spy%20on%20executed%20operations%20via%20the%20onCall%20prop.

Reply:
@OzzieOrca I couldn't really figure it out I don't see a way to test it.  If you are OK with it we can just leave the test as is.

Reply:
I don't think we really need an expectation for a config option... I could help you figure it out if you want though.